# Reg2ST: Recognizing potential patterns from gene expression for spatial transcriptomics prediction

## Overview
Spatial transcriptomics is able to acquire cellular gene expression while retaining spatial location. It is often accompanied by matched hematoxylin and eosin (H&E) stained histology whole-slice images (WSI). This retention of spatial information is critical for studying key issues in cell biology, developmental biology, neurobiology, and tumor biology. However, conventional sequencing technologies are costly and time-consuming, limiting the development of spatial transcriptomics research. Recently deep learning methods have been widely applied to spatial transcriptomics prediction, but there are some problems with the existing methods. To tackle the problem, we develop Reg2ST, a deep learning model to learn potential patterns in gene expression and apply the learned patterns to histology images for spatial transcriptomics prediction. For each spot, we first use a gene encoder and an image encoder to extract features from the gene expression and histology images, respectively, and design the contrastive learning loss to draw these two features closer together by treating the spatial transcriptomics and the histology images as expressions of different modal data for the same spot. Subsequently,  we use the image features to obtain the predicted gene features and embed the position information into the image features. The predicted gene features and the integrated image features are jointly input into the cross-attention layer for feature fusion. Finally, in the dynamic graph neural network module, the global and local relationships between spots are preceived. Spatial transcriptomics are predicted by following the zero-inflated negative binomial distribution. Comprehensive benchmark evaluations using human breast cancer and cutaneous squamous cell carcinoma datasets demonstrate the superior performance of Reg2ST for spatial gene expression prediction.

## Framework of Reg2ST
![model](https://github.com/Holly-Wang/Reg2ST/blob/main/model.png)

## Results
![her2st result](https://github.com/Holly-Wang/Reg2ST/blob/main/res_her2st.png)

![cscc result](https://github.com/Holly-Wang/Reg2ST/blob/main/res_cscc.png)
## Dataset
In this study, we use spatial transcriptomics datasets same as the previous study, including two datasets: the human HER2-positive breast cancer dataset (HER2+) and the human cutaneous squamous cell carcinoma (cSCC) dataset. Both datasets include histology images, gene expression  and coordinates of spots. Specifically, the HER2+ breast tumor dataset contains 36 sections from 8 patients. To ensure the uniformity and reliability of data, we retained 32 sections from 7 patients. Each of them contains at least 180 spots. These data comprehensively record the spatial gene expression patterns of HER2+ breast tumors and provide an important resource for studying the heterogeneity of the tumor microenvironment. The cSCC dataset contains human cutaneous squamous cell carcinoma tissue samples, including 12 sections from 4 patients, with each patient contributing 3 sections. These sections were generated by the 10x Visium platform, providing high-resolution gene expression data.

